arrows/serialize/json/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ set( headers_public
   metadata.h
   object_track_set.h
   object_track_state.h
+  serialize_metadata.h
```

Comment:
This serializer used by dump_klv is only built if JSON serialization is selected in CMake, so there is a subtle dependency on this CMake option if you want to dump metadata to a file.

Reply:
There is a `core` implementation of csv metadata writing which was supposed to address this issue. It looks like it is still [solely internal](https://kwgitlab.kitware.com/computer-vision/KWIVER/-/blob/nga-klv/master/arrows/core/serialize_metadata_csv.cxx) though. I plan to open a WIP PR for that for bookkeeping purposes, but I think this is a much higher priority.